docs/source/commands.rst

```diff
@@ -28,6 +28,10 @@ Basic ``forecast`` usage:
 * :code:`n_folds` - number of folds to use in the backtest for prediction interval estimation. By default equals to 3.
 * :code:`return_components` - whether to estimate forecast components
 * :code:`start_timestamp` - timestamp with the starting point of forecast.
+* :code:`estimate_n_folds` - whether to estimate the number of folds from data. Works only when prediction intervals are enabled. Requires :code:`context_size` parameter set in pipeline config.
+
```

Comment:
Yes, the line below references a bit different config (a field in pipeline config). So a blank line is needed to visually separate these sections.

Reply:
In that case we should separate this blocks by its name I think. May be create a new subsection for this. Because this parameter isn't a parameter of forecast config. As I understood, it is a parameter of pipeline config.

Reply:
Added separate sections like  `forecast`/`backtest`